src/app/Component/Admin/edit-manager/edit-manager.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Toast, ToastrService } from 'ngx-toastr';
+import { manager } from 'src/app/Models/manager';
+import { NavbarService } from 'src/app/Services/navbar.service';
+import { ServicesService } from 'src/app/Services/services.service';
+import { NavbarComponent } from 'src/app/common/navbar/navbar.component';
+
+@Component({
+  selector: 'app-edit-manager',
+  templateUrl: './edit-manager.component.html',
+  styleUrls: ['./edit-manager.component.scss']
+})
+export class EditManagerComponent {
+  id;
+  manager=new manager();
+  empRoles;
+  employees;
+  employeesMgrs;
+  departments;
+  constructor(private service:ServicesService,private route: ActivatedRoute,private Nav:NavbarService,private toastr:ToastrService,private router:Router){
+
+  }
+  ngOnInit(){
```

Comment:
Instead of filling all Dropdown at page load/initialize, fill them sequentially.
Like on Page load, fill Department List,
On selection of Department, fill list of Manager & Employee (On the base of Role)